deployments/helm-chart/values.yaml

```
@@ -433,6 +433,19 @@ prometheus:
   ## Configures the HTTP scheme used.
   scheme: http
 
+serviceInsight:
+  ## Expose NGINX Plus Service Insight endpoint.
+  create: true
```

This shouldn't be defaulted to true, this is the template for both OSS and Plus, defaulting this Plus only feature to true will cause deployment failures for OSS 
```suggestion
  create: false
```